packages/dd-trace/src/config.js

```diff
@@ -397,6 +397,12 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       true
     )
 
+    const DD_IAST_REDACTION_ENABLED = coalesce(
+      iastOptions && iastOptions.redactionEnabled,
+      process.env.DD_IAST_REDACTION_ENABLED && isTrue(process.env.DD_IAST_REDACTION_ENABLED),
```

Comment:
```suggestion
      isTrue(process.env.DD_IAST_REDACTION_ENABLED),
```

Reply:
Actually, I think this could even just be `!isFalse(process.env.DD_IAST_REDACTION_ENABLED)` and drop the default value if the default is true.